src/modules/ipfix/IpfixReceiverDtlsUdpIpV4.cpp

```diff
@@ -315,7 +315,11 @@ int IpfixReceiverDtlsUdpIpV4::DtlsConnection::accept() {
     char buf[512];
     ret = SSL_accept(ssl);
     if (SSL_get_shared_ciphers(ssl,buf,sizeof buf) != NULL)
-	DPRINTF("Shared ciphers:%s",buf);
+#ifdef DEBUG
```

Comment:
Can you briefly explain why we need this DEBUG switch here?

Reply:
DPRINTF does nothing when DEBUG is not defined, so for that case I had to add {} to silence gcc.

Reply:
Actually I should probably surround DPRINTF with {} and remove the #ifdef

Reply:
Should we redefine the `msg` macro so that it outputs `{}` if the log level is too low? This would also apply to the previous change in `src/common/openssl/SSLCTXWrapper.cpp`

Reply:
Unfortunately, this will not work, because, for example in the case found in src/common/openssl/SSLCTXWrapper.cpp, the macro will expand as follows:
if (...)
   {};     <- here {} will be considered the "then" body, which is followed by an ";"
else  <- so here the compiler will give an "‘else’ without a previous ‘if’" error
   ...